pkgs/top-level/all-packages.nix

```diff
@@ -11763,7 +11763,7 @@ with pkgs;
 
   darwin = let
     apple-source-releases = callPackage ../os-specific/darwin/apple-source-releases { };
-  in apple-source-releases // rec {
+  in recurseIntoAttrs (apple-source-releases // rec {
```

Comment:
apple-source-releases should be okay to build, right? Maybe we can just use that?

Questionable ones might be:
- usr-include
- osx_private_sdk
- xcode
- cf-private

(they are probably already being built just not listed on hydra because they are dependencies of different things)